### ISSUE TITLE
False Positive Likes...

### ISSUE THREAD

**Issue by ghost** [InstaPy/InstaPy#1842]
##  **TL;DR:** 
Log/script outputs false positives on liked images.

## Long description:
I am running InstaPy on digital ocean via crontab and noticed follows and like backs slowed down greatly.
Checking the logs everything seems to be in order. There are zero error messages and the output is good. Image Liked, person followed, etc.
I decided to check the image my user was suppose to have liked.... and to my surprise the user did not take any action on the image.
Oddly I can run the same script from my local with positive results.
Hoping someone can point me in the right direction as I am puzzled.
Thanks in advance for any help.

## Script:

```python
from instapy import InstaPy
import random

numbxxx = 30
numberToLike = random.randint(3, 6)

tagsToLike = ([xxx])

tagsToLike = random.sample(tagsToLike, numberToLike)

tagsToLike.append('xxx')

print(tagsToLike)

session = InstaPy(username='xxx',
                  password='xxx',
                  bypass_suspicious_attempt=True,
                  headless_browser=True)

session.login()

session.set_dont_include(['xxx'])

session.like_by_tags(tagsToLike, amount=numbxxx, media='Photo')

session.end()
```

**Comment by ghost** [comment]
This is still happening oddly.
The script is running, looks successful, but viewing the asset no like took place. 
False positives...

Any help is much appreciated.

**Comment by ghost** [comment]
Digital Ocean IP is blocked from Instagram.
